scarb/src/core/package/id.rs

```diff
@@ -136,7 +142,8 @@ impl<'de> Deserialize<'de> for PackageId {
 
 impl fmt::Display for PackageId {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "{} v{}", self.name, self.version)?;
+        let name = self.name.to_string().replace(TEST_PACKAGE_PREFIX, "");
+        write!(f, "{} v{}", name, self.version)?;
```

Comment:
I have mixed feelings on this. This loses potentially important information. Can we drop this change?